lib/api/gitlab.js

```diff
@@ -0,0 +1,296 @@
+const logger = require('winston');
+const glGot = require('gl-got');
+
+const config = {};
+
+module.exports = {
+  initRepo,
+  // Search
+  findFilePaths,
+  // Branch
+  branchExists,
+  getBranchPr,
+  // issue
+  addAssignees,
+  addReviewers,
+  addLabels,
+  // PR
+  findPr,
+  checkForClosedPr,
+  createPr,
+  getPr,
+  updatePr,
+  // file
+  commitFileToBranch,
+  getFile,
+  getFileContent,
+  getFileJson,
+};
+
+// Initialize GitLab by getting base branch
+async function initRepo(repoName, token) {
```

Comment:
Line _45_ of `index.js` does not pass the `token` from `config.token` to this function:

```javascript
await api.initRepo(config.repository);
```